src/main/kotlin/com/corradowaver/bot/listeners/MemberJoinedGuildListener.kt

```diff
@@ -19,14 +19,16 @@ class MemberJoinedGuildListener(
   private val tempDirPath = createTempDirectory("guild-girl-greetings")
 
   override fun onGuildVoiceJoin(event: GuildVoiceJoinEvent) {
-    val randomS3Greeting = s3Manager.getRandomGreeting()
-    val file = saveS3ObjectToTempDir(randomS3Greeting)
-    val voiceChannel = event.member.voiceState?.channel
-    MusicHandler().loadAndPlay(
-      event.guild,
-      file.path,
-      voiceChannel
-    )
+    if (!event.member.user.isBot) {
+      val randomS3Greeting = s3Manager.getRandomGreeting()
+      val file = saveS3ObjectToTempDir(randomS3Greeting)
```

Comment:
Bruh will temp files ever be deleted lol cringe? I know it will, bro. Sorry, bro. Good work!

Reply:
yonzoo can we have more structured reviews in the future please?! Thank you! 😇

Reply:
THank you!!!!!!!!!!!!!!!!!!!!! Temp files are deleted automatically in some cringe music events listeners! 😃

Reply:
sure m8, but you are not a collaborator bro, how did u come here?

Reply:
very goood brotheereresreresrsrsrsrsrsrs

Reply:
bro u blind?